### PR TITLE
casebundle changes for cloudpak cert 2.3.0

### PR DESCRIPTION
**Pull request by Jainbrt**

### CODE REVIEW

File: cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/prereqs.yaml [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
-      #  distribution: ibmCloudPrivate
-      #  semver: '>=1.11.3'
-      #kubernetes:
-      #  distribution: kubernetes
-      #  semver: '>=1.14'
-      #openshift:
-      #  distribution: openshift
-      #  semver: '>=4.2'
-    k8sResources:  {}
+    k8sDistros:
```
Comment (maroche15) [inline cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/prereqs.yaml:4]: We had the discussion that the prerequisite definitions here mean you need to use them and when you don't, produce this error:
```
[ERROR]: The prereq item kubernetes, is declared but not used in any actions for any inventory item (CASEProcessingError)
[ERROR]: The prereq item openshift, is declared but not used in any actions for any inventory item (CASEProcessingError)
```

If you guys will not be supporting CASE based installation, I would revert the changes made to this file so that the `casectl verify` command could execute successfully.

File: cloudpak/.travis.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    on:
+      all_branches: true
+  # Publish tagged release
+  - provider: releases
+    skip_cleanup: true
+    api_key: $GITHUB_TOKEN
+    file_glob: true
+    file: repo/stable/*
+    on:
+      tags: true
```
Comment (maroche15) [inline cloudpak/.travis.yml:47]: I am not sure that this file will end up needing to be checked in here, but I don't think it should cause an issue if it does go in with the rest of the changes.

**Review by maroche15 — COMMENTED** [review]
These changes look good to me overall so far. One area that will need to be updated to work with these changes are the test cases within cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/tests. These test cases are executing commands using files that no longer exist.

**Comment by Jainbrt** [comment]
> 
> 
> These changes look good to me overall so far. One area that will need to be updated to work with these changes are the test cases within cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/tests. These test cases are executing commands using files that no longer exist.

done these changes

File: cloudpak/.travis.yml (hunk 2 above)
Comment (Jainbrt) [reply cloudpak/.travis.yml:47]: Sure, If you want, I can remove that. Please let me know.

**Comment by maroche15** [comment]
Changes look good to me and we should be good to merge.

**Review by deeghuge — APPROVED** [review]
LGTM
